### PR TITLE
Ansible cruise

### PR DESCRIPTION
**Pull request by githubamid**

### CODE REVIEW

**Comment by githubamid** [comment]
Up Ansible cruise

File: Dmitry_Khomich/08.Ansible.Spinup/roles/spinup/tasks/redhat_http.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    r_var: "{{ out.stdout_lines }}"
+#- name: Print element of list
+#  debug:
+#    msg: "{{ r_var|length }}"
+- set_fact:
+    value: "{{ sudo_var if ( r_var|length > 1) else nosudo_var }}"
+- name: Print Permissions
+  debug:
+    msg: "{{ value }}"
+- name: Remove Git on RedHat family if it installed
```
Comment (pluhin) [inline Dmitry_Khomich/08.Ansible.Spinup/roles/spinup/tasks/redhat_http.yaml]: Do not use shell, better to use module

File: Dmitry_Khomich/08.Ansible.Spinup/roles/spinup/tasks/redhat_http.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+- name: Git shell install RedHat
```
Comment (pluhin) [inline Dmitry_Khomich/08.Ansible.Spinup/roles/spinup/tasks/redhat_http.yaml]: Do not use shell, better to use module

File: Dmitry_Khomich/08.Ansible.Spinup/roles/spinup/tasks/main.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+- name: If you get "msg- 200" connection to docker hub registry check!!!
+  debug:
+    msg: "{{ out.status }}"
+- name: Info about hosts
+  set_fact:
+   mount: "{{ ansible_mounts | first }}"
+#- name: Info about mount
+#  debug:
+#    msg: "{{ mount }}"
+- set_fact: disk_total="{{ mount.size_total }}"
```
Comment (pluhin) [inline Dmitry_Khomich/08.Ansible.Spinup/roles/spinup/tasks/main.yaml]: Can you merge in one task here

File: Dmitry_Khomich/08.Ansible.Spinup/roles/spinup/tasks/debian_http.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
+- name: Git shell install Debian
+  shell: apt install -y git
```
Comment (pluhin) [inline Dmitry_Khomich/08.Ansible.Spinup/roles/spinup/tasks/debian_http.yaml]: module here

File: Dmitry_Khomich/08.Ansible.Spinup/roles/spinup/tasks/alpine.yaml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+- name: Git shell install Alpine
+  shell: apk add git
```
Comment (pluhin) [inline Dmitry_Khomich/08.Ansible.Spinup/roles/spinup/tasks/alpine.yaml]: Module

File: Dmitry_Khomich/08.Ansible.Spinup/roles/spinup/molecule/default/molecule.yml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  - name: alpine
+    image: alpine:3
+provisioner:
+  name: ansible
+  lint:
+    name: ansible-lint
+    enabled: true
+    options:
+      x:
+        - '201'
```
Comment (pluhin) [inline Dmitry_Khomich/08.Ansible.Spinup/roles/spinup/molecule/default/molecule.yml]: Why to many ignores????

File: Dmitry_Khomich/08.Ansible.Spinup/group_vars/uhosts/uhosts.yaml [hunk 7]
```diff
@@ -0,0 +1,4 @@
+ansible_ssh_user: root
+ansible_ssh_pass: QwertY_13
+ansible_ssh_bridge_pass: 23mnZujif4rzU
```
Comment (pluhin) [inline Dmitry_Khomich/08.Ansible.Spinup/group_vars/uhosts/uhosts.yaml]: Do not keep pass here, use vault

File: Dmitry_Khomich/08.Ansible.Spinup/group_vars/jump/jump.yaml [hunk 8]
```diff
@@ -0,0 +1,2 @@
+ansible_ssh_user: jump_sa
+ansible_ssh_pass: 23mnZujif4rzU
```
Comment (pluhin) [inline Dmitry_Khomich/08.Ansible.Spinup/group_vars/jump/jump.yaml]: Do not keep pass here, use vault

File: Dmitry_Khomich/08.Ansible.Spinup/group_vars/infra/infra.yaml [hunk 9]
```diff
@@ -0,0 +1,4 @@
+ansible_ssh_user: root
+ansible_ssh_pass: QwertY_13
+ansible_ssh_bridge_pass: 23mnZujif4rzU
```
Comment (pluhin) [inline Dmitry_Khomich/08.Ansible.Spinup/group_vars/infra/infra.yaml]: Vault!!!

File: Dmitry_Khomich/08.Ansible.Spinup/group_vars/all/all_remote.yaml [hunk 10]
```diff
@@ -0,0 +1,6 @@
+ansible_ssh_user: root
+ansible_ssh_pass: QwertY_13
+#ansible_ssh_common_args: '-o ProxyCommand="sshpass -p 23mnZujif4rzU ssh -W %h:%p 178.124.206.48"'
+#ansible_ssh_common_args: '-o ProxyCommand="ssh -W %h:%p root@178.124.206.48"'
+ansible_ssh_bridge_pass: 23mnZujif4rzU
```
Comment (pluhin) [inline Dmitry_Khomich/08.Ansible.Spinup/group_vars/all/all_remote.yaml]: Vault

File: Dmitry_Khomich/08.Ansible.Spinup/group_vars/all/all.yaml [hunk 11]
```diff
@@ -0,0 +1,4 @@
+ansible_ssh_user: root
+ansible_ssh_pass: QwertY_13
+ansible_ssh_bridge_pass: 23mnZujif4rzU
```
Comment (pluhin) [inline Dmitry_Khomich/08.Ansible.Spinup/group_vars/all/all.yaml]: You need to declare user/pass just once, do not repeat same, use variables instead, as one of solution

**Comment by pluhin** [comment]
Any updates?

**Comment by githubamid** [comment]
What are the issues in my Ansible.cruise task - ?
     I did a minor update to pay attention to this task

**Comment by githubamid** [comment]
Vault password for Ansible.Spinup Version 2.0 like on working hosts

**Pull request merged**